src/suparnaNJune20/Bank.java

```diff
@@ -0,0 +1,72 @@
+package suparnaNJune20;
+
+public class Bank {
+
+	static double atmBal=100000;
+	static int debitCnt, creditCnt, printBillCnt;
+	int debit,credit, printBill;
+	
+	void debitTransaction(int amt, String name) {
+		if (amt>atmBal)	{
+			System.out.println("No Sufficient BAl ");
+			return;
+		}
+		else {
+			atmBal-=amt;
+			debit++;
+			debitCnt++;
+			System.out.println(amt+ " Rs debited sucesfully by " + name);
+		}
+	}
+		void  creditTransaction (int amt, String name) {
+			 atmBal+=amt;
+				System.out.println(amt+ " Rs credited sucesfully by"+ name);
+				credit++;
+				creditCnt++;
+	}		
+		void  printBillTransaction() {
+			System.out.println(" Current Balance in Atm :"+atmBal);
+			printBill++;
+			printBillCnt++;
+		}
+	
+	void individualTransactionSumary(String custName) {
+		System.out.println("\n\n Total No of  trnasaction done by  "+custName +" are "+ (debit+credit+ printBill));
+		System.out.println("Total No of Credit transaction " + credit);
+		System.out.println("Total No of Debit transaction " + debit);
+		System.out.println("Total No of PrintBill transaction "+ printBill);
+    }
+	
+
+	void allTransactionSumary() {
+		System.out.println("\n\nTotal No of  trnasaction done by all custome are  "+ (debitCnt+creditCnt+ printBillCnt));
+		System.out.println("Total No of Credit transaction " + creditCnt);
+		System.out.println("Total No of Debit transaction " + debitCnt);
+		System.out.println("Total No of PrintBill transaction "+ printBillCnt);
+	}
+	public static void main(String[] args) {
+		// TODO Auto-generated method stub
+		Bank customer1 = new Bank();
+		Bank customer2 = new Bank();
+		System.out.println("Current ATM bal : "+customer1.atmBal );
+		for (int i = 40; i<45; i++)
```

Comment:
When we write for a look range between 40 to 45, it will be a confusing part for the end-user that why you have selected this range. So here, for loop is not needed.

Reply:
ddone